src/pages/tutor-profile/TutorProfile.jsx

```diff
@@ -45,7 +52,9 @@ const TutorProfile = () => {
       <ProfileInfo myRole={userRole} userData={response} />
       <CompleteProfileBlock data={response} profileItems={profileItems} />
       <AboutTutorBlock />
-      <VideoPresentation />
+      {(isTutor || (!isTutor && response.videoLink?.student)) && (
```

Comment:
Please consider creating variable `shouldShowPresentation` and move this logic up to avoid messing up our `jsx` code

Reply:
Thanks for the suggestion, I moved it.